propagation/propagators_test.go

```diff
@@ -53,6 +53,16 @@ func mustSpanIDFromHex(s string) (t trace.SpanID) {
 	return
 }
 
+func TestBytesMapCarrier(t *testing.T) {
+	carrier := make(propagation.BytesMapCarrier)
+	carrier.Set("foo", "bar")
+	carrier.Set("baz", "qux")
+
+	assert.Equal(t, carrier.Get("foo"), "bar")
+	assert.Equal(t, carrier.Get("baz"), "qux")
+	assert.Equal(t, carrier.Keys(), []string{"foo", "baz"})
```

Comment:
Map iteration order isn't guaranteed, so the result will need to be sorted or another set of checks devised.  You could assert it's length 2 and contains both "foo" and "bar".

Reply:
Sorry, fixed it.